### DISCUSSION TITLE
UI Login Problems

### DISCUSSION THREAD

**Discussion by MrYellowOwl** [amidaware/tacticalrmm#1220] [Q&A]
**Server Info (please complete the following information):**
 - OS: [e.g. Ubuntu 20.04 ]
 - Browser: [chrome & Edge]
 - RMM Version (as shown in top left of web UI):

**Installation Method:**
  - [X ] Standard
  - [ ] Docker

**Agent Info (please complete the following information): agents work
- Agent version (as shown in the 'Summary' tab of the agent from web UI): latese
- Agent OS: win11

The issue I am dealing with is that i am unable to login to the ui I have restarted the server and updated everything on the server side however still I am unable to login to the ui.

**Comment by MrYellowOwl** [comment]
Backend is offline (network error)

**Reply by MrYellowOwl** [reply]
it was an ssl issue thank you for the help

**Reply by dinger1986** [reply]
Lol it's always ssl (or dns)

**Comment by ninjamonkey198206** [comment]
Do this:
1) Install dnsutils
```bash
sudo apt install dnsutils
```
2) Switch to tactical user (assuming you used tactical as user)
```bash
sudo su - tactical
```
3) Run troubleshoot_server.sh, verify that all services are running and all dns entries are good
```bash
cd /rmm
chmod +x troubleshoot_server.sh
./troubleshoot_server.sh
```

**Reply by MrYellowOwl** [reply]
it was an ssl issue thank you for the help

**Reply by jeffdurden** [reply]
how did you resolve?  I have the exact same issue.

**Reply by dinger1986** [reply]
Do you have an ssl cert installed?

**Comment by MrYellowOwl** [comment]


**Reply by dinger1986** [reply]
install script? where you installing?

**Reply by MrYellowOwl** [reply]
it was the troubleshoot script

**Reply by dinger1986** [reply]
Ah ok. Doesn't look like Postgres is running.

**Comment by jeffdurden** [comment]
I got it figured out. I was trying to use it without having port 443 forwarded though the firewall to the server. Apparently this is required.

Jeff Durden
